SWLOR.Game.Server/Feature/ShipModuleDefinition/HypermatterInjectorModuleDefinition.cs

```diff
@@ -51,7 +51,7 @@ private void HypermatterInjector(string tag, string name, string shortName, stri
                 })
```

Comment:
This was already here, but there's a typo for the short name on line 21. "FUel" -> "Fuel" Please correct this as part of this PR.